core/iwasm/libraries/debug-engine/handler.c

```diff
@@ -175,6 +175,19 @@ process_wasm_global(WASMGDBServer *server, char *args)
     os_mutex_unlock(&tmpbuf_lock);
 }
 
+/* TODO: let server send an empty/error reply.
+   Original issue: 4265
+   Not tested yet, but it should work.
+ */
+static void
+fail(WASMGDBServer *server, const char *err)
```

Comment:
Maybe it's better to rename it to `send_error_reply` or something, fail as a function name is a little strange since WAMR often uses fail as a goto label